<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OneVino</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/bt/image/bg.png);</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="lab_bt">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>377</y>
      <width>360</width>
      <height>358</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="meeting_bt">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>377</y>
      <width>360</width>
      <height>358</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="visitor_bt">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>342</y>
      <width>360</width>
      <height>428</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="reg_button">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>40</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Industry_bt">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>340</y>
      <width>360</width>
      <height>428</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
